### PR TITLE
:bug: Fix tag rank input 

### PR DESCRIPTION
**Pull request by ibolton336**
https://issues.redhat.com/browse/MTA-1195

### CODE REVIEW

File: client/src/app/pages/controls/tags/components/tag-category-form.tsx [hunk 1]
```diff
@@ -189,7 +189,12 @@ export const TagCategoryForm: React.FC<TagCategoryFormProps> = ({
             onMinus={() => {
               onChange((value || 0) - 1);
             }}
-            onChange={() => onChange}
+            onChange={(event) => {
+              const target = event?.target as HTMLInputElement;
+              if (target) {
+                onChange(target.valueAsNumber);
+              }
+            }}
```
Comment (sjd78) [inline client/src/app/pages/controls/tags/components/tag-category-form.tsx:197]:
```suggestion
            onChange={onChange}
```

Would that work?

Comment (sjd78) [reply client/src/app/pages/controls/tags/components/tag-category-form.tsx:197]: Yeah ok, that's crazy.

Looking at the next component in the file, your original change looks reasonable.

Comment (ibolton336) [reply client/src/app/pages/controls/tags/components/tag-category-form.tsx:197]: Haha! I think we avoid this with the TextInput and other components interfacing with react-hook-form by creating a wrapper and using some fancy typescript extension of the passed in onChange handler type. @mturley Could probably explain it better than I.

Comment (sjd78) [reply client/src/app/pages/controls/tags/components/tag-category-form.tsx:197]: Eventually I will get react-hook-form and the wrapper components around PF fields loaded up in my brain.  Until then it'll just be an adventure.

**Review by mturley — APPROVED** [review]

**Review by gitdallas — APPROVED** [review]

**Pull request merged**